.env.development

```diff
@@ -1 +1,4 @@
-REACT_APP_PG_SERVICE_URL=http://localhost:8081/data.json
+REACT_APP_PG_SERVICE_URL=https://vu00h9m96c.execute-api.us-east-2.amazonaws.com/staging/cairo-staging
+#REACT_APP_PG_SERVICE_URL=http://192.168.3.54:8100/data.json
+REACT_APP_STARKNET_ALPHA_SERVICE_URL=https://alpha4.starknet.io
+REACT_APP_SHARP_SERVICE_URL=https://testnet.provingservice.io
```

Comment:
?

Reply:
You shouldn't push the manipulated `.env.development`
You can use instead `.env.development.local` which is git ignored.